dask_jobqueue/core.py

```diff
@@ -344,6 +344,11 @@ def worker_spec(self):
             "memory": self.worker_process_memory,
         }
 
+    @property
```

Comment:
Don't include this here, but leave it in #282.

Reply:
This is required for the tests to pass, it's related to the name change upstream.